server/database/users.go

```diff
@@ -35,3 +35,21 @@ func GetPLayerIdForUser(db *sql.DB, userId int64, player string) (int64, error)
 	}
 	return id, nil
 }
+
+func IsObjectOwnedByPlayer(db *sql.DB, playerId int64, objectId int64) (bool, error) {
+	ownedObjects := make(map[int64]struct{})
+	rows, err := db.Query("select `id` from t_object where owner = ?", playerId)
```

Comment:
I think this should check specifically for alive ships. Currently, this imho allows players to give commands to wrecks.
@malytomas opinion?

Reply:
to get all ships owned by a player:
`select t_object.id from t_object join t_ship on t_ship.id = t_object.id where t_object.owner = ?`

Reply:
Created another function with the provided query: GetPlayerOwnedShips